disnake/flags.py

```diff
@@ -1055,10 +1313,8 @@ def __init__(self, **kwargs: bool):
     @classmethod
     def all(cls) -> Self:
         """A factory method that creates a :class:`MemberCacheFlags` with everything enabled."""
-        bits = max(cls.VALID_FLAGS.values()).bit_length()
-        value = (1 << bits) - 1
         self = cls.__new__(cls)
-        self.value = value
+        self.value = all_flags_value(cls.VALID_FLAGS)
```

Comment:
missing change in `__init__` above

Reply:
https://github.com/DisnakeDev/disnake/pull/605/commits/6cddd65dec9bef4e00350ebcb61a7df7d704be5d